app/src/main/java/nu/yona/app/api/manager/impl/AuthenticateManagerImpl.java

```diff
@@ -810,27 +813,27 @@ public User getUser()
 		return authenticateDao.getUser();
 	}
 
+	@Override
+	public JSONObject getStoredUserObjectFromDB()
```

Comment:
This one we don't need, see my comment on ``getStoredUserObjectFromDB``

Reply:
Removed.